### ISSUE TITLE
etcd added to an existing cluster is not able to join existing etcd cluster.

### ISSUE THREAD

**Issue by sangeethah** [rancher/rke#45]
rke version v0.0.4-dev

Steps to reproduce the problem:
Create a cluster with following configuration:
host1 in etcd and controlplane
host2 in controlplane
host3 in worker

Update cluster (using cluster up command)  by making host2 in etcd as well
etcd service gets deployed in host2 successfully but it is not able join existing cluster.

```
INFO[0022] [etcd] Building up Etcd Plane..              
INFO[0025] [etcd] Container etcd is already running on host [sangee-test-1] 
INFO[0029] [etcd] Successfully pulled etcd image on host [sangee-test-2] 
INFO[0029] [etcd] Successfully started etcd container on host [sangee-test-2] 
INFO[0029] [etcd] Successfully started Etcd Plane..   
```

```
/ # etcdctl member list
client: etcd cluster is unavailable or misconfigured; error #0: dial tcp 127.0.0.1:4001: getsockopt: connection refused
; error #1: client: endpoint http://127.0.0.1:2379 exceeded header timeout

/ # 
```

**Comment by galal-hussein** [comment]
A fix added to raise error if a node added/removed during upgrade

**Comment by galal-hussein** [comment]
The fix can be tested with v0.0.5-dev

**Comment by sangeethah** [comment]
Tested with rke version v0.0.5-dev
Following error message is presented to the user when trying to add/remove etcd nodes to existing k8s cluster
```
FATA[0001] Adding or removing Etcd nodes is not supported 
```
